index.html

```diff
@@ -1,279 +1,274 @@
+                <li class="language">Selenium</li>
+            </ul>
+            <ul class="Language_skill_list3">
+                <li class="Language_skill_bullet"><img class="border_line" src="./images/Rectangle57.png" alt="circle_icon">Skills</li>
+                <li class="language">Codekit</li>
+                <li class="language">Github</li>
+                <li class="language">Codepen</li>
+                <li class="language">Gitlab</li>
+                <li class="language">Terminal</li>
+            </ul>
         </div>
     </section>
-    <section>
-      <div class="formpage_content_container">
-         <p class="formpage_content"><strong>
-          I'm always interested in hearing about new projects, so if you'd like
-          to chat please get in touch.
-          </strong>
-        </p>
-      </div>
-        <form class="contact_me_form"
-          action="https://formspree.io/f/xoqbwpby"
-          method="POST">
-          <label for="name" hidden>Name</label>
-          <input
-            type="text"
-            name="Name"
-            id="name"
-            class="inputs"
-            placeholder="Full names here"
-            maxlength="50"
-            required
-          /><br>
-          <label for="email" hidden>Email</label>
-          <input
-            type="email"
-            name="Email"
-            id="email"
-            class="inputs"
-            placeholder="abcd@email.com"
-            required
-          />
-          <label for="message" hidden>Message</label>
-          <textarea
-            name="Message"
-            id="message"
-            placeholder="Enter text here"
-            class="inputs"
-            id=""
-            cols="30"
-            rows="3"
-            maxlength="500"
-            required
-          ></textarea>
-          <button type="submit" name="button" class="button">
-            Get in touch
-          </button>
+
+    <section class="form_section">
+        <h3 class="form_text">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h2>
+    
+        <form class="contact_me_form" action="https://formspree.io/f/xoqbwpby" method="post">
+            <label hidden for="text"> Full Name </label>
+            <input class="form_input form_input_height" type="text" placeholder="Enter full names" maxlength="30" required>
+            <hr>
+            <label hidden for="text"> Email </label>
+            <input class="form_input form_input_height" type="email" placeholder="abcd@form.com" required>
+            <hr>
+            <label hidden for="text"> Enter your text</label>
+            <textarea class="form_input form_input_email" name="" id="" cols="30" rows="5" placeholder="Leave us a message" maxlength="500" required></textarea> <br>
+            <button type="submit" class="get_in_touch_button">Get in touch</button>
+            <hr class="border_line_form">
         </form>
```

Comment:
- [ ] Please stick to the Figma tempalte as much as possible because it make your portfolio much more attractive and its good for your sharpening skills. the back for the contact form is white. 
## your design 👇
![contactst](https://user-images.githubusercontent.com/66992901/194669995-b448216e-2e2a-49c2-a0ea-d646f896c65c.PNG)
## Figma design👇
![contactfigma](https://user-images.githubusercontent.com/66992901/194670026-cb54afb8-e842-48f5-8542-af38c4b5e044.PNG)

Reply:
-Like the previous reviewer suggested, your contact form doesn't match the design given in the Figma template, please try to fix this 🙏 🙏 . Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

## Your design:
<img width="948" alt="okonta contact" src="https://user-images.githubusercontent.com/73607512/194699403-72c30ff2-75ae-4f3d-b7b0-5a3be337bd50.PNG">